### PR TITLE
Us 2/add new trips

### PR DESCRIPTION
**Pull request by malgosiam**
After submitting the form, the user is currently redirected to the "My Trips" page.

### CODE REVIEW

File: components/NewTripForm/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+import StyledButton from "../StyledButton";
+import Link from "next/link";
+import { useState } from "react";
+
+export default function NewTripForm({ onSubmit }) {
+  const [startDate, setStartDate] = useState("");
+  const [endDate, setEndDate] = useState("");
+
```
Comment (augustinanders) [inline components/NewTripForm/index.js:9]: You could use just one State here, maybe sth. like this: 

```
const [formState, setFormState] = useState({
    startDate: "",
    endDate: "",
  })
```

**Review by augustinanders — APPROVED** [review]
Looks good to me. Your code is very readable. 🚀

left a comment below.

**Comment by malgosiam** [comment]
I am very pleased to read that. Thank you for your review Augustin!

File: components/NewTripForm/index.js (hunk 1 above)
Comment (malgosiam) [reply components/NewTripForm/index.js:9]: Thanks for your insight! This is an interesting topic. After reading the React documentation and other related resources, I will leave this code flat for now, but I will keep an eye on it!

**Pull request merged**